Rendera/include/display.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef DISPLAY_HPP
+#define DISPLAY_HPP
+
+#include "ray.hpp"
+#include "color.hpp"
+class Hitinfo{
+    bool ishitting;
+    float dist;
+    vec3 intersection_pt;
+    vec3 normal_pt;
+    ray ray_;
+    color shade;
+};
+class Display{
+    public:
+        int img_width;
```

Comment:
The image height and width should probably be unsigned ints because you can't have a negative number of pixels. It would be even better to use an explicitly sized type like `uint32_t`, because the sizes of `int`s (unsigned or otherwise) can vary from machine to machine.